pipeline-automation-scripts/pipeline_automatation_uc1/main.py

```diff
@@ -0,0 +1,336 @@
+import requests
+import logging
+import time
+import sys
+import json
+import google.auth
+import google.auth.exceptions
+import google.auth.transport.requests
+import constants
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+def create_compute_profile(api_endpoint, credential_token):
+    ''' Creates CDF compute profiles
+    
+        parameters:
+
+        api_endpoint(string): Endpoint of cdf instance
+
+        credential_token(string): credential token
+    '''
+    bearer_token = str('Bearer ' + credential_token).strip()
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': bearer_token
+    }
+    
+    for json_url,profile in zip(constants.json_path,constants.profiles):
```

Comment:
please add comments to each step describing what it does

Reply:
Fixed